pages/task/read/content/confirm.js

```diff
@@ -4,17 +4,25 @@ module.exports = {
       title: 'Confirm decision',
       decision: {
         title: 'Your decision',
+        label: 'Endorsement',
         change: 'Change'
       },
       declaration: {
-        title: 'Declaration',
-        text: `By endorsing this application, I agree that:
-           * I have the authority of the establishment licence holder.
-        `
+        title: 'Declaration'
       },
       link: {
-        exit: 'Exit'
+        exit: 'Save and exit'
```

Comment:
I may be wrong, but I think this link text is used (or should be used) on all forms, so it ideally should be pulled up a few levels to the base content

Reply:
Makes sense, but sounds like a lane change - let's create a ticket.

Reply:
It is used on a lot of forms, but also it's fictional functionality so we should avoid it IMO.

Reply:
good point...

Reply:
We might as well label the button "click this and we'll lose your data".